backends/text-to-speech/linux/linux-text-to-speech.h

```diff
@@ -59,7 +60,8 @@ class SpeechDispatcherManager : public Common::TextToSpeechManager {
 	SpeechDispatcherManager();
 	virtual ~SpeechDispatcherManager() override;
 
-	virtual bool say(Common::String str, Action action, Common::String charset = "") override;
+	virtual bool say(const Common::U32String &str, Action action, Common::String charset = "") override;
+	virtual bool say(const Common::String &str, Action action, Common::String charset = "") override;
```

Comment:
This one doesn't need to be virtual. It could be implemented just in the base class converting the string to UTF-32 and calling the one above.